etc/delphix-platform/upgrade/unpack-image

```diff
@@ -86,4 +95,6 @@ mv "$UNPACK_DIR" "$UPDATE_DIR/$VERSION" ||
 rm -f "$UPDATE_DIR/latest" || die "failed to remove 'latest' symlink"
 ln -s "$VERSION" "$UPDATE_DIR/latest" || die "failed to create 'latest' symlink"
 
+report_progress_inc 40 "Unpacking successful."
```

Comment:
Should this report 100? 40 was already reported above...?

Reply:
Thanks for the catch. I forgot to re-push my local changes to my forked remote.